mne/beamformer/tests/test_rap_music.py

```diff
@@ -87,30 +86,6 @@ def simu_data(evoked, forward, noise_cov, n_dipoles, times):
 
 
 def _check_dipoles(dipoles, fwd, stc, evoked, residual=None):
-    src = fwd['src']
-    pos1 = fwd['source_rr'][np.where(src[0]['vertno'] ==
-                                     stc.vertices[0])]
-    pos2 = fwd['source_rr'][np.where(src[1]['vertno'] ==
-                                     stc.vertices[1])[0] +
-                            len(src[0]['vertno'])]
-
-    # Check the position of the two dipoles
-    assert_true(dipoles[0].pos[0] in np.array([pos1, pos2]))
-    assert_true(dipoles[1].pos[0] in np.array([pos1, pos2]))
-
-    ori1 = fwd['source_nn'][np.where(src[0]['vertno'] ==
-                                     stc.vertices[0])[0]][0]
-    ori2 = fwd['source_nn'][np.where(src[1]['vertno'] ==
-                                     stc.vertices[1])[0] +
-                            len(src[0]['vertno'])][0]
-
-    # Check the orientation of the dipoles
-    assert_true(np.max(np.abs(np.dot(dipoles[0].ori[0],
-                                     np.array([ori1, ori2]).T))) > 0.99)
-
-    assert_true(np.max(np.abs(np.dot(dipoles[1].ori[0],
-                                     np.array([ori1, ori2]).T))) > 0.99)
```

Comment:
But the `DipoleFixed` object doesn't have the same attributes as the `Dipole` object. What are the tests I could do or the attributes of `DipoleFixed` that I could use?